static/src/stylesheets/module/commercial/_creatives.scss

```diff
@@ -847,9 +847,7 @@
 }
 
 .creative__background-parent {
-    transform: translate3d(0, 0, 0);
-    will-change: transform;
-    contain: strict;
+    contain: size layout style;
```

Comment:
yes, because `paint` makes this element a containing block for its fixed positioned child, which cancels the intended effect.